var/spack/repos/builtin/packages/sensei/package.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Sensei(CMakePackage):
+    """SENSEI is a platform for scalable in-situ analysis and visualization.
+    Its design motto is 'Write once, run everywhere', this means that once
+    the application is instrumented with SENSEI it can use existing and
+    future analysis backends. Existing backends include: Paraview/Catalyst,
+    Visit/Libsim, ADIOS, Python scripts, and so on."""
+
+    homepage = "https://sensei-insitu.org"
+    url      = "https://gitlab.kitware.com/sensei/sensei/-/archive/v2.1.1/sensei-v2.1.1.tar.gz"
+    git      = "https://gitlab.kitware.com/sensei/sensei.git"
+
+    version('master', branch='master')
+    version('2.1.1', sha256='8a27ebf133fef00a59e4b29433762e6560bf20214072de7808836eb668bb5687')
+    version('2.1.0', sha256='b7af21a25523cf6cd8934d797471b75ca32881166625d71f24b5c8b6d727ca99')
+    version('2.0.0', sha256='df48eab035e1acdd8edf5159955c05306f9ca48117effacc4a6b77c3fb24f62b')
+    version('1.1.0', sha256='e5a4ba691573ff6c7b0d4793665e218ee5868ebcc0198915d1f16a4b7b92a368')
+    version('1.0.0', sha256='bdcb03c56b51f2795ec5a7e85a5abb01d473d192fac50f2a8bf2608cc3564ff8')
+
+    variant('build_type', default='RelWithDebInfo', description='CMake build type', values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))
+    variant('sencore', default=True, description='Enables the SENSEI core library')
+    variant('catalyst', default=True, description='Build with ParaView-Catalyst support')
+    variant('libsim', default=False, description='Build with VisIt-Libsim support')
+    variant('vtkio', default=True, description='Enable adaptors to write to VTK XML format')
+    variant('adios', default=False, description='Enable ADIOS adaptors and endpoints')
+    variant('python', default=False, description='Enable Python bindings')
+    variant('miniapps', default=True, description='Enable the parallel 3D and oscillators miniapps')
+
+    depends_on("paraview@5.5:5.6", when="+catalyst")
```

Comment:
Is `paraview` sufficient or does it need to be `paraview+python`?

Reply:
@chuckatkins It's a good point, it should actually be: paraview+python+mpi+hdf5. Because when compiling Paraview for Sensei we don't want it to compile Python, mpi4py, or HDF5. I still need to figure out which paraview version should it be. Paraview recipe specifies that version 5.6 can work only with Python 3, but SENSEI's constraint is Python 2. So that leaves only Paraview 5.5.x.